sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidSqlValidator.java

```diff
@@ -136,6 +136,24 @@ public void validateWindow(SqlNode windowOrId, SqlValidatorScope scope, @Nullabl
       );
     }
 
+    boolean hasBounds = lowerBound != null || upperBound != null;
```

Comment:
Can we add a negative test here to indicate what kind of queries are we restricting so that the user can get an idea. Also good candidate to indicate in the docs as well as NTILE does not support framing

Reply:
note that I don't see any rational use case how someone could utilize `NTILE` with framing  :D